rust/tools/authorization-logic/src/ast.rs

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2021 The Raksha Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+use serde::*;
```

Comment:
Has this been formatted with rustfmt? I would have expected a blank line after the license header.

Reply:
It has actually. I also just tried running it again and it did not add a space. I copied over the config file from the Oak repo. Am I doing something wrong or does the config file actually need to be changed somehow?

Reply:
In Oak we are not using block quotes for the license headers. I don't know whether that perhaps makes a difference.